src/xenia/cpu/backend/x64/x64_seq_vector.cc

```diff
@@ -594,7 +594,7 @@ struct VECTOR_ADD
 
                     e.vpsrad(e.xmm2, e.xmm1, 31);
                     e.vpxord(e.xmm2, e.xmm2, e.GetXmmConstPtr(XMMSignMaskI32));
-                    e.vpblendmd(dest | saturate, e.xmm1, e.xmm2); 
```

Comment:
I know this change was most likely done by the formatter but it does pollute the commit so probably worth reverting (@gibbed can decide for sure)

Reply:
Ah yeah, no problem to revert it. The CI linting check was failing because of it, so I figured I'd leave it in at the time. Working on the coding style changes, will push a commit soon.
(I've mostly done C code in the past and I'm not super familiar with C++ language features, so all feedback and code review is very much appreciated.)

Reply:
I removed this commit for now, so the CI linting will probably fail, but I can add it back if desired.